### PR TITLE
feat(graphql): add ability to search persons by any document type

### PR DESCRIPTION
**Pull request by progsmile**
connects #4200

### CODE REVIEW

File: apps/graphql/lib/graphql_web/schema/person_types.ex [hunk 1]
```diff
@@ -230,9 +233,9 @@ defmodule GraphQLWeb.Schema.PersonTypes do
 
   defp transform_persons_query_filter(%{arguments: %{filter: filter} = arguments} = resolution, _params) do
     filter =
-      case get_in(filter, [:documents, :number]) do
+      case Map.get(filter, :birth_date) do
         nil -> filter
-        number -> %{filter | documents: %{type: "PASSPORT", number: number}}
+        _ -> Map.merge(filter, %{is_active: true, status: "active"})
```
Comment (PavelVesnin) [inline apps/graphql/lib/graphql_web/schema/person_types.ex]: can we use module attribute for `active` status?

Comment (progsmile) [reply apps/graphql/lib/graphql_web/schema/person_types.ex]: sure

Comment (PavelVesnin) [reply apps/graphql/lib/graphql_web/schema/person_types.ex]: let's do it

File: apps/graphql/lib/graphql_web/resolvers/person_resolver.ex [hunk 2]
```diff
@@ -10,7 +10,11 @@ defmodule GraphQLWeb.Resolvers.PersonResolver do
   alias Core.Persons.Person
   alias GraphQLWeb.Loaders.OPS
 
+  @person_status_active "active"
+
   def list_persons(%{filter: filter, order_by: order_by} = args, _resolution) do
+    filter = Enum.concat(filter, [{:status, :equal, @person_status_active}, {:is_active, :equal, true}])
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/resolvers/person_resolver.ex]:
```suggestion
    filter = [{:status, :equal, @person_status_active}, {:is_active, :equal, true} | filter]
```

**Pull request merged**